tools/cldr-apps/src/main/java/org/unicode/cldr/web/DBUtils.java

```diff
@@ -815,6 +819,13 @@ public final Connection getDBConnection() {
      * @return
      */
     public final Connection getAConnection() {
+        if (DEBUG_DB_OPEN) {
+            final String myClass = DBUtils.class.getName();
+            final StackTraceElement callerElement = StackTracker
+                .firstCallerMatching(
+                    (StackTraceElement s) -> !s.getClassName().equals(myClass));
+            System.out.println("DB OPEN: " + callerElement);
+        }
```

Comment:
@btangmu might be able to rewrite this with #1220 to use a log level where this code is only executed if the right level of detail is enabled